src/inventory/gilded_rose.js

```diff
@@ -22,49 +22,80 @@ const items = [
 updateQuality(items);
 */
 export function updateQuality(items) {
+
   for (var i = 0; i < items.length; i++) {
-    if (items[i].name != 'Aged Brie' && items[i].name != 'Backstage passes to a TAFKAL80ETC concert') {
-      if (items[i].quality > 0) {
-        if (items[i].name != 'Sulfuras, Hand of Ragnaros') {
-          items[i].quality = items[i].quality - 1
+
+   //------VARIABLES-----//
```

Comment:
It's fine to keep, but it's not a common practice since we can see variables by let, const, and var declarations. It's also might be potentially confusing if variable declarations are done outside of this block because it implies that all variables are near this annotation. If it helps you in this individual project, that's totally fine but I would recommend training your eyes to also spot variables using the variable declarations since annotations the location of variables like this is extremely uncommon in codebases.

Reply:
I removed it, I see what you mean and want to nip it in the bud.